<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="legoset_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>34</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>34</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="legoset_new_pushButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Lego Set hinzufügen</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Images/Images.qrc">
           <normaloff>:/Images/legoset_new.png</normaloff>:/Images/legoset_new.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="legoset_delete_pushButton">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Lego Set hinzufügen</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Images/Images.qrc">
           <normaloff>:/Images/legoset_delete.png</normaloff>:/Images/legoset_delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="legoset_tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Projekt</string>
    </property>
    <addaction name="action_load"/>
    <addaction name="action_save"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_load">
   <property name="icon">
    <iconset resource="Images/Images.qrc">
     <normaloff>:/Images/project_open.png</normaloff>:/Images/project_open.png</iconset>
   </property>
   <property name="text">
    <string>Laden</string>
   </property>
   <property name="toolTip">
    <string>Projekt Laden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="Images/Images.qrc">
     <normaloff>:/Images/project_close.png</normaloff>:/Images/project_close.png</iconset>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
   <property name="toolTip">
    <string>Proejkt Schließen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="Images/Images.qrc">
     <normaloff>:/Images/app_exit.png</normaloff>:/Images/app_exit.png</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="Images/Images.qrc">
     <normaloff>:/Images/project_save.png</normaloff>:/Images/project_save.png</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="toolTip">
    <string>Projekt Speichern</string>
   </property>
  </action>
  <action name="action_legoset_new">
   <property name="icon">
    <iconset resource="Images/Images.qrc">
     <normaloff>:/Images/legoset_new.png</normaloff>:/Images/legoset_new.png</iconset>
   </property>
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="action_legoset_delete">
   <property name="icon">
    <iconset resource="Images/Images.qrc">
     <normaloff>:/Images/legoset_delete.png</normaloff>:/Images/legoset_delete.png</iconset>
   </property>
   <property name="text">
    <string>Löschen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images/Images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
